client/test/e2e/inventory/inventory_list.spec.js

```diff
@@ -0,0 +1,76 @@
+/* global element, by, inject, browser */
+'use strict';
+
+const uuid   = require('node-uuid');
+const chai   = require('chai');
+const expect = chai.expect;
+
+const FU = require('../shared/FormUtils');
+const helpers = require('../shared/helpers');
+const components = require('../shared/components');
+
+helpers.configure(chai);
+
+describe.only('Inventory List ::', () => {
```

Comment:
Please remember to remove `.only()` from your tests.  It is very frustrating to have to check this in ever PR. :(